### PR TITLE
docs: Form disabled version

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
ci 挂了

File: components/drawer/__tests__/__snapshots__/demo-extend.test.tsx.snap [hunk 1]
```diff
@@ -1767,7 +1767,7 @@ HTMLCollection [
                                                     </div>
                                                   </td>
                                                   <td
-                                                    class="ant-picker-cell ant-picker-cell-in-view ant-picker-cell-today"
+                                                    class="ant-picker-cell ant-picker-cell-in-view"
```
Comment (afc163) [inline components/drawer/__tests__/__snapshots__/demo-extend.test.tsx.snap:1770]: 为啥这个会变。

**Comment by afc163** [comment]
冲突了

**Comment by MadCcc** [comment]
我 clean 一下

**Pull request merged**